contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -154,55 +153,117 @@ Surveys are great for tracking customer satisfaction scores like Net Promoter Sc
   <ComparisonRow column1={true} column2={false} feature="API mode" description="Create surveys via the API" />
 </ComparisonTable>
 
-### Integrations
+- Surveys are great for tracking customer satisfaction scores like Net Promoter Score (NPS), customer satisfaction (CSAT), and customer effort score (CES).
+
+- We include [survey templates](/templates?filter=type&value=survey) for all these and more helping you get started fast. Read our [guide comparing NPS, CSAT and CES](/product-engineers/nps-vs-csat-vs-ces) to see which one you should use and when.
 
-PostHog and Amplitude both offer a wide range of integrations, sources and destinations. PostHog offers more integrations with developer tools, such as [Sentry](/blog/posthog-vs-sentry), Zapier and [Zendesk](/tutorials/zendesk-session-replays), while Amplitude has more integrations with marketing and advertising platforms. See our [data pipeline docs](/docs/cdp) for a full list of PostHog integrations.
+## Price comparison
+
+### Free tier
```

Comment:
Hmm do we want to lean in very heavily here? Im not sure. I think we're starting to come off as a bit cheap if we focus on free tier too much. I worry this will scare big companies which expect to pay for prodcuts

Reply:
Fair. We cover the free usage in each product section, so we probably don't need to double up here.